docs/kbs.yaml

```diff
@@ -71,19 +71,37 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorInformation'
 
-  /resources/key/<key_id>:
+  /resources/{repository}/{type}/{tag}:
```

Comment:
Also, could we use that more generic resource definition for tokens as well:

`GET /resource/token/token_tag`